Gemfile.lock

```diff
@@ -2,14 +2,15 @@ GEM
   remote: https://rubygems.org/
```

Comment:
My Ruby ignorance is probably showing, but I wonder whether what appears to be a lock file should be part of the project repository.